gtsfm/scene_optimizer.py

```diff
@@ -60,7 +60,8 @@ def __init__(
         feature_extractor: FeatureExtractor,
         two_view_estimator: TwoViewEstimator,
         multiview_optimizer: MultiViewOptimizer,
-        save_viz: bool,
+        save_image_pair_viz: bool,
```

Comment:
change to `save_two_view_correspondences_viz`?

Reply:
Sure, thanks

Reply:
fixed